cg/src/domains/management/stats/stats.service.js

```diff
@@ -0,0 +1,43 @@
+const { db } = require('../../../db');
+
+class StatsService {
+  constructor(database = db) {
+    this.db = database;
+  }
+
+  async stats() {
+    const [result] = await db('subjects')
+      .select(db.raw('count(subjects.id) as consented'))
+      .join('subject_keys', 'subjects.id', 'subject_keys.subject_id');
```

Comment:
We could have one query instead of two to left join on keys and count when not null (use `COALESCE` with `CASE` or something like it), but I don't expect a huge performance gain.